var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -97,6 +98,12 @@ def make_defs(self):
             'FC={0}'.format(spack_f77),
             'MAKE_NO_J=1'
         ]
+        # invoke make with the correct TARGET for aarch64
+        if 'aarch64' in spack.architecture.sys_type():
+            make_defs += [
+                'TARGET=PILEDRIVER',
+                'TARGET=ARMV8'
+        ]
```

Comment:
Flake8 tests are complaining about this line:
```
var/spack/repos/builtin/packages/openblas/package.py:106: [E122] continuation line missing indentation or outdented
```
If you indent this `]` one more level you should be good to go. You can run `spack flake8` to test this locally.